.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -999,6 +934,85 @@ protected double GetPointValue(List<string> matchStrs)
             return ret;
         }
 
+        protected int GetSplitIndex(List<string> fracWords)

```

Comment:
Change to private please

Reply:
Could you also add documentation to this method instead? Will help explain what split index is

Reply:
Done.